Octokit/Clients/IRepositoryCommentsClient.cs

```diff
@@ -18,17 +18,17 @@ public interface IRepositoryCommentsClient
         /// <remarks>http://developer.github.com/v3/repos/comments/#get-a-single-commit-comment</remarks>
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
-        /// <param name="number">The comment id</param>
+        /// <param name="commentId">The comment id</param>
```

Comment:
Not sure if we want to rename this to `commentId`. Perhaps the maintainers should chime in on this one.

Reply:
It is a breaking change fix anyway. I would like to see the maintainer here to reduce the id/number confusion around the code.